### PR TITLE
Bump core and olf version + add matpower and PSSE converters

### PR DESCRIPTION
**Pull request by MioRtia**
Signed-off-by: RALAMBOTIANA MIORA <miora.ralambotiana@rte-france.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Update versions + add matpower and psse converters

**Other information:**
Do not squash the commits

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -59,6 +59,15 @@
                             <file>${project.basedir}/resources/CGMES/CGMES_v2.4.15_MicroGridTestConfiguration_BD_v2/MicroGridTestConfiguration_EQ_BD.xml</file>
                             <file>${project.basedir}/resources/CGMES/CGMES_v2.4.15_MicroGridTestConfiguration_BD_v2/MicroGridTestConfiguration_TP_BD.xml</file>
                             <file>${project.basedir}/resources/IEEE/ieee300cdf.txt</file>
+                            <file>${project.basedir}/resources/MATPOWER/ieee9.json</file>
```
Comment (geofjamg) [inline pom.xml:62]: I would not add these files as it is not really matpower files but ref file for tests

**Review by mathbagu — COMMENTED** [review]
@MioRtia You should increment the version of the project to avoid another pull request.

File: pom.xml (hunk 1 above)
Comment (mathbagu) [reply pom.xml:62]: I agree: maybe we can convert few files from https://github.com/MATPOWER/matpower/blob/master/data/case6515rte.m (like RTE file)?

Comment (geofjamg) [reply pom.xml:62]: Why not but this is probably not necessary to provide test cases for any of our importer.

**Pull request merged**